src/containers/guest-home-page/login-form/LoginForm.jsx

```diff
@@ -31,6 +35,17 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  useEffect(() => {
+    const { email: emailError, password: passwordError } = errors
+    const { email: emailData, password: passwordData } = data
+
+    if (!emailError && !passwordError && emailData && passwordData) {
```

Comment:
as i remember, button should be disabled only when fields are empty

Reply:
Yeah, you are right 😅. Fixed it